docs/features/autoupdater.rst

```diff
@@ -50,6 +50,35 @@ The header contains the following information:
   the number of days over which the update should be stretched out after ``DATE``. Nodes will calculate a probability
   based on the time left to determine when to update.
 
+Signing images
+--------------
+
+As noted above, manifest files can be signed by an arbitrary amount of ECDSA keys.
+The amount of valid signatures required to have the autoupdater acknowledge an image is configured using the ``site.conf``.
```

Comment:
```suggestion
The amount of valid signatures required to have the autoupdater accept an image is configured using the ``site.conf``.
```

Same term is used here https://gluon.readthedocs.io/en/latest/user/site.html#configuration

But no strong opinion

Also it might make sense to actually link to that page instead of just saying `site.conf`?

Reply:
resolved in af3723a2771ffd0e9d4d9de9f7507d7ab5bae060, thanks.